taxi/views.py

```diff
@@ -1,3 +1,18 @@
 from django.shortcuts import render
+from django.http import HttpRequest, HttpResponse
```

Comment:
Consider removing the unused import `render` from `django.shortcuts` since it is not used in this file. This will help keep the code clean and avoid unnecessary imports.